src/models/disassemblyoutput.cpp

```diff
@@ -49,9 +53,31 @@ static QVector<DisassemblyOutput::DisassemblyLine> objdumpParse(const QByteArray
                 qWarning() << "unhandled asm line format:" << asmLine;
                 continue;
             }
+
+            const auto leftBracketIndex = asmLine.indexOf(QLatin1Char('<'));
```

Comment:
extract this code out into a separate function then call it below, i..e something like

```
struct LinkedFunction
{
    QString name;
    int offset = 0;
};

LinkedFunction extractLinkedFunction(const QString &disassembly);
...
disassemblyLines.push_back({addr, asmLine, extractLinkedFunction(asmLine)});
```